### PR TITLE
Make golangci-lint happy

### PR DESCRIPTION
**Pull request by qbarrand**
Fixes [this problem](https://github.com/openshift-psap/special-resource-operator/runs/4162376901?check_suite_focus=true).

### CODE REVIEW

File: pkg/kernel/kernel_test.go [hunk 1]
```diff
@@ -39,6 +40,7 @@ func TestSetAffineAttributes(t *testing.T) {
 			obj := newObj(kind, objNewName)
 
 			err := SetAffineAttributes(obj, kernelFullVersion, operatingSystemMajorMinor)
+			require.NoError(t, err)
```
Comment (ybettan) [inline pkg/kernel/kernel_test.go:43]: Not familiar with that code, can you explain what this method is supposed to do and why it is fixing the CI?

Comment (qbarrand) [reply pkg/kernel/kernel_test.go:43]: This is a unit test that was introduced in #121. We were initializing `err` to the result of `SetAffineAttributes(obj, kernelFullVersion, operatingSystemMajorMinor)`, but never actually checking the contents of `err`.  
This PR fixes this by making sure that `err` is not an error, using [require.NoError](https://pkg.go.dev/github.com/stretchr/testify/require#NoError).

**Pull request merged**